### PR TITLE
Added Clarifying Steps

### PR DESCRIPTION
**Pull request by graceful-coder**
Changed the name of branin.arff to file0 per the previous instruction in the flow. Added clarifications.

Fixes # .

Changes proposed in this PR:

- 
- 
-

### CODE REVIEW

File: READMEs/main-flow.md [hunk 1]
```diff
@@ -102,8 +102,25 @@ file_name = os.path.join(asset_dir, "file0")
 Let's check that the file is downloaded. In a new console:
 
 ```console
-cd my_project/datafile.did:op:0xAf07...
-ls branin.arff
+cd my_project/datafile.did:op:*
+vi file0
```
Comment (mariacarmina) [inline READMEs/main-flow.md:106]: can we use `cat` here please?

**Review by mariacarmina — COMMENTED** [review]
just a minor tweak

File: READMEs/main-flow.md (hunk 1 above)
Comment (graceful-coder) [reply READMEs/main-flow.md:106]: Cat opens the end of the file, however. That's why I went with vi. Is there a reason to prefer cat?

Comment (mariacarmina) [reply READMEs/main-flow.md:106]: maybe some of the users do not have experience with vim and `cat` is used only for listing the content of a specific file. I personally use it often.

**Review by mariacarmina — APPROVED** [review]
LGTM

**Review by calina-c — APPROVED** [review]

**Pull request merged**